src/pdfflow/pflow.py

```diff
@@ -488,3 +488,82 @@ def py_alphasQ(self, arr_q):
 
         # Perform the actual computation
         return self.alphasQ(a_q)
+
+    def trace(self):
+        """
+        Builds all the needed graph in advance
+        of interpolations
+        """
+        print('Building tf.Graph ...')
+        x = []
+        q2 = []
+
+        #x points are equal for all the subgrids
+        xmin = float(tf.math.exp(self.subgrids[0].log_xmin))
+
+        q2min = float(tf.math.exp(self.subgrids[0].log_q2min))
```

Comment:
Instead of casting it to float, I think it would be better to do `.numpy()` when possible.